sushy/redfish_interop_profiles/ServiceRoot.v1_0_0.json

```diff
@@ -0,0 +1,66 @@
+{
+    "@Redfish.Copyright": "Copyright (c) 2020 Dell Inc. or its subsidiaries.",
+    "@Redfish.License": "Apache License, Version 2.0. For full text, see link: http://www.apache.org/licenses/LICENSE-2.0",
+    "SchemaDefinition": "RedfishInteroperabilityProfile.v1_0_0",
```

Comment:
We should change the version to v1_3_1 to pick up the latest changes to the profile schema.

Reply:
Done.